app/views/idv/shared/_verify.html.erb

```diff
@@ -4,84 +4,94 @@ locals:
   remote_identity_proofing - true if we're doing RIDP, false if we're doing IPP
   step_url - generator for step URLs
 %>
-<% title t('titles.doc_auth.verify_info') %>
+<% title t('titles.idv.verify_info') %>
 
-<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.verify')) %>
+<%= render PageHeadingComponent.new.with_content(t('headings.verify')) %>
 <div class='margin-top-4 margin-bottom-2'>
   <div class="grid-row grid-gap grid-gap-2 padding-bottom-1">
-    <div class='grid-col-fill'>
+    <dl class="grid-col-fill margin-bottom-0">
```

Comment:
Description lists have a default bottom margin of 2 so I wanted to override it to match the div that was here previously